dev-tools/build.sh

```diff
@@ -8,15 +8,15 @@ composer config platform.php 2> /dev/null || composer config platform.php 5.3.6
 composer require --no-update symfony/polyfill-mbstring
 
 # install package deps without dev-deps / remove already installed dev-deps
-composer update --no-interaction --no-progress --no-dev --prefer-stable
+composer update --no-interaction --no-progress --prefer-stable
 composer info -D | sort
 
-# install box2 globally
-composer global show kherge/box -q || composer global require --no-interaction --no-progress kherge/box:^2.7
+if [ ! -f dev-tools/vendor ]; then
```

Comment:
the fact that folder exists doesn't prove that box is there
` composer show -d dev-tools humbug/box || composer install -d dev-tools` would check it properly
or always run `composer install -d dev-tools` without checking.